src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -324,7 +324,6 @@ class queue_cb {
     void stop_aio_worker(void)
     {
         if (qAioBatchEn) {
-            std::lock_guard<std::mutex> lk(reqLock);
```

Comment:
really?   I wouldn't have left this one; I think synchronization and a memory point is missing if you remove this.  In my experience, the thread routine will never see qExit changed.

Reply:
I see. I guess the value is cached. Let me fix it.